### PR TITLE
feat(editing-support): add keymaps to refactoring-nvim

### PR DESCRIPTION
**Pull request by jay-babu**

### CODE REVIEW

File: lua/astrocommunity/editing-support/refactoring-nvim/refactoring-nvim.lua [hunk 1]
```diff
@@ -3,4 +3,97 @@ return {
...
+      },
+      desc = "Extract Block To File",
+    },
+    {
+      "<leader>ri",
+      [[ <Cmd>lua require('refactoring').refactor('Inline Variable')<CR>]],
+      { silent = true, expr = false },
+      mode = { "n" },
+      desc = "Inline Variable",
+    },
```
Comment (mehalter) [inline lua/astrocommunity/editing-support/refactoring-nvim/refactoring-nvim.lua:67]:
```suggestion
    {
      "<leader>rb",
      function() require('refactoring').refactor('Extract Block') end,
      { silent = true, expr = false },
      mode = {
        "n",
      },
      desc = "Extract Block",
    },
    {
      "<leader>rbf",
      function() require('refactoring').refactor('Extract Block To File') end,
      { silent = true, expr = false },
      mode = {
        "n",
      },
      desc = "Extract Block To File",
    },
    {
      "<leader>ri",
      function() require('refactoring').refactor('Inline Variable') end,
      { silent = true, expr = false },
      mode = { "n" },
      desc = "Inline Variable",
    },
```

These should just be lua functions

**Review by mehalter — CHANGES_REQUESTED** [review]
The visual mode mappings make sense to be `vimscript` strings, but I let a revision for a few that can be moved to just lua functions

File: lua/astrocommunity/editing-support/refactoring-nvim/refactoring-nvim.lua (hunk 1 above)
Comment (jay-babu) [reply lua/astrocommunity/editing-support/refactoring-nvim/refactoring-nvim.lua]: Sure, I mostly just copied pasted

**Review by mehalter — APPROVED** [review]
I went ahead and fixed the formatting. Thanks!

**Comment by jay-babu** [comment]
> I went ahead and fixed the formatting. Thanks!

Thanks. I can't do anything from my work computer

**Pull request merged**